src/main/java/org/spongepowered/api/item/ItemType.java

```diff
@@ -61,6 +61,13 @@
     @Override
     String getName();
 
+    /**
+     * Gets the {@link ItemGroup} that contains this item.
+     *
+     * @return The item group or {@link Optional#empty()} otherwise
+     */
+    Optional<ItemGroup> getItemGroup();
```

Comment:
An item can be in multiple groups.

Reply:
Quite possibly needs to be a `Collection<ItemGroup>`, since `Collections.empty()` is easy to call.